ddtrace/llmobs/_llmobs.py

```diff
@@ -507,6 +534,7 @@ def annotate(
             cls._tag_metrics(span, metrics)
         if tags is not None:
             cls._tag_span_tags(span, tags)
+        return LLMObs.export_span(span)
```

Comment:
This might improve the dev experience a bit by giving uses their exported eval data right after they set it on a span.

Also there's the tricky part where the span has to be annotated before it is exported so it's enriched with the correct data 

Wasn't originally planning on doing this but thought it would be nice to have -- thoughts?